app/src/main/java/com/nextcloud/talk/conversationcreation/ConversationCreationViewModel.kt

```diff
@@ -32,13 +32,20 @@ class ConversationCreationViewModel @Inject constructor(
     val selectedParticipants: StateFlow<List<AutocompleteUser>> = _selectedParticipants
     private val roomViewState = MutableStateFlow<RoomUIState>(RoomUIState.None)
 
+    private val selectedImageUri = MutableStateFlow<Uri?>(null)
+    val selectedImageUriState: StateFlow<Uri?> = selectedImageUri
```

Comment:
mutable property should be prefixed with an underscore (_) and public val should be the same name without underscore:

private val _selectedImageUri = ...
val selectedImageUri StateFlow<Uri?> = _selectedImageUri